### DISCUSSION TITLE
SuperSlicer Bridging Issue

### DISCUSSION THREAD

**Discussion by djngnear** [supermerill/SuperSlicer#2430] [Q&A]
How do I get SuperSlicer to recognize a thin extrusion as a bridged extrusion? I've noticed at times when perimeters are too thin the bridging is not detected. This instance is from the latest 2.4 release but I've seen it happen before this.
Thanks.

3MF-
https://drive.google.com/file/d/12NFTb9CzxSwSgKCxOd3H0bn5m7F5zgp1/view?usp=sharing

![SS Bridge Issue](https://user-images.githubusercontent.com/87901234/154869880-25ddc2d8-f70c-4d5f-a97a-03a1ecf8fc0a.JPG)
![SS Bridge Issue 2](https://user-images.githubusercontent.com/87901234/154869923-be369b0c-f964-448e-aee5-f54fe0b31f55.JPG)

**Comment by supermerill** [comment]
currently not possible. #238
